Release notes for 25.06 (#790)
NVIDIA/nv-ingest: docs/docs/extraction/releasenotes-nv-ingest.md

    NeMo Retriever extraction is also known as NVIDIA Ingest and nv-ingest.

## Release 25.06

The NeMo Retriever extraction 25.06 release focuses on accuracy improvements and feature expansions, including the following:

- Improve reranker accuracy.
- Upgrade Python version from 3.10 to 3.12
- Helm deployment now has similar throughput performance to docker deployment.
- Add support for the latest version of the OpenAI API.

> - Add MIG support. For details, see [Enable NVIDIA GPU MIG](https://github.com/nkmcalli/nv-ingest/blob/main/helm/README.md#enable-nvidia-gpu-mig).

Reviewer: ```suggestion
- Add MIG support. For details, see [Enable NVIDIA GPU MIG](https://github.com/NVIDIA/nv-ingest/blob/25.6.1/helm/README.md#enable-nvidia-gpu-mig).
```

Author: nkmcalli fixed, keeping main

> - Add time slicing support. For details, see [Enable GPU time-slicing](https://github.com/nkmcalli/nv-ingest/blob/main/helm/README.md#enabling-gpu-time-slicing).

Reviewer: ```suggestion
- Add time slicing support. For details, see [Enable GPU time-slicing](https://github.com/NVIDIA/nv-ingest/blob/25.6.1/helm/README.md#enabling-gpu-time-slicing).
```

Author: nkmcalli fixed, keeping main

> - Add support for RIVA NIM for optional audio extraction. For details, see [helm/values.yaml](https://github.com/NVIDIA/nv-ingest/blob/main/helm/values.yaml).

Reviewer: ```suggestion
- Add support for RIVA NIM for optional audio extraction. For details, see [helm/values.yaml](https://github.com/NVIDIA/nv-ingest/blob/25.6.1/helm/values.yaml).
```

Author: keeping main

> - New notebook for [How to add metadata to your documents and filter searches](https://github.com/nkmcalli/nv-ingest/blob/main/examples/metadata_and_filtered_search.ipynb).

Reviewer: ```suggestion
- New notebook for [How to add metadata to your documents and filter searches](https://github.com/NVIDIA/nv-ingest/blob/25.6.1/examples/metadata_and_filtered_search.ipynb).
```
